Source/WebKit/Shared/API/c/WKSharedAPICast.h

```diff
@@ -656,7 +654,7 @@ inline WebCore::ContextMenuAction toImpl(WKContextMenuItemTag tag)
     case kWKContextMenuItemTagOther:
         return WebCore::ContextMenuItemTagOther;
     case kWKContextMenuItemTagSearchInSpotlight:
-        return WebCore::ContextMenuItemTagSearchInSpotlight;
+        return WebCore::ContextMenuItemTagNoAction;
```

Comment:
Will it ever be possible to fully get rid of these?

Reply:
we probably already could, I was just being conservative.